packages/web/src/feature_switches.js

```diff
@@ -1,27 +1,26 @@
 import axios from 'axios';
 import { createContext, useState } from 'react';
-import { parseToNewAPI, url } from './Global';
+import { featureSwitchesURL } from './Global';
 
 // Easily disable sections globally
 
 // The following represent the initial global values of the switches
 // These values will be replaced after a successful execution of a fetch
 const FEATURE_SWITCHES = {
     jobs: false,
-    team: true,
     regForm: false,
+    team: true,
 };
 
-const featureSwitchesURL = `${url}/v2/fswitches`;
-
 export const loadFeatureSwitches = () => {
-    const result = axios(parseToNewAPI(featureSwitchesURL), {
+    const result = axios(featureSwitchesURL, {
         method: 'get',
     })
         .then(res => {
             return res.data.documents;
         })
         .catch(() => {
+            window.alert('API is not responding!');
```

Comment:
This approach is incorrect. Feature switches have default state. If no response is received, their state is not being updated. 

This update will result in a user facing error message which is not what we want.

Reply:
The API is not responding message should be displayed only within the admin panel

Reply:
yup i know i will do it in another way